uk.co.compucorp.civicrm.hrsampledata/tests/phpunit/CRM/HRSampleData/Importer/AbsenceTypeTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Class CRM_HRSampleData_Importer_AbsenceTypeTest
+ *
+ * @group headless
+ */
+class CRM_HRSampleData_CSVProcessor_AbsenceTypeTest extends CRM_HRSampleData_BaseCSVProcessorTest {
+
+  public function setUp() {
+    $this->rows = [];
+    $this->rows[] = $this->importHeadersFixture();
+  }
+
+  public function testProcess() {
+
+    $absenceType = $this->apiGet('AbsenceType', ['title' => '2016']);
+    $this->assertEmpty($absenceType);
+
+    $this->rows[] = [
+      'Annual Leave',
+      1,
+      '#151D2C',
+      1,
+      0,
+      3,
+      0,
+      1,
+      20.00,
+      1,
+      1,
+      0,
+      0,
+      1,
+      5.00,
+      12,
+      2,
+      0,
+      1
+    ];
+
+    $this->runProcessor('CRM_HRSampleData_Importer_AbsenceType', $this->rows);
+
+    $absenceType = $this->apiGet('AbsenceType', ['type' => 'Annual Leave']);
+
+    foreach($this->rows[0] as $index => $fieldName) {
+      $this->assertEquals(
+        $this->rows[1][$index],
+        $absenceType[$fieldName],
+        "The value of {$fieldName} was expected to be {$this->rows[1][$index]}, but it is {$absenceType[$fieldName]}"
```

Comment:
Was this message for debugging purpose because I noticed its not in the tests for some of the other L&A entities?

Reply:
It was for debugging and to make the error messages better. Without it, we would only see a message like "Expected value to be 1 but 5 was found" which doesn't mean anything when we do assertions inside of a loop. I'll add the same to the other tests